File: day6/java/AgeCalculator.java
import java.util.Scanner;
import java.time.LocalDate;
import java.time.Period;
import java.time.format.DateTimeFormatter;

	public class AgeCalculator {
	
public static void main(String[] args){
	Scanner input = new Scanner(System.in);
	System.out.print("Enter your birth date: ");
	String dateOfBirth = input.next();
       System.out.println(getAge(dateOfBirth));

}
		public  static int getAge(String dateOfBirth){
			DateTimeFormatter formatter = DateTimeFormatter.ofPattern("[dd/MM/yyyy][dd-MM-yyyy][dd/MMM/yyyy][dd MM yyyy]");
			LocalDate localDate = LocalDate.parse(dateOfBirth, formatter);
			LocalDate currentDate = LocalDate.now();
			Period period = Period.between(localDate, currentDate);
				
				
				return period.getYears();
		

	

	}
}

File: day6/python/AgeCalculator.py
from datetime import datetime

def age_calculator(birth_date_day):
  birth_date = datetime.strptime(birth_date_day, "%d-%m-%Y")
  today = datetime.today()
  age = today.year - birth_date.year
  return age

if __name__ == "__main__":
 
  birth_date_day = input("Enter your birth date : ")
  age = age_calculator(birth_date_day)
  print(age)

